test/cert_utils.go

```diff
@@ -131,7 +132,12 @@ func GenerateLeafCert(subject string, oidcIssuer string, parentTemplate *x509.Ce
 			Id:       asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 1},
 			Critical: false,
 			Value:    []byte(oidcIssuer),
-		}},
+		},
```

Comment:
nit: To avoid having to update all places where GenerateLeafCert is called, can we create a new function like GenerateLeafCertWithGitHubOIDs or something?

Reply:
that makes a lot of sense, thank you, updated.